packages/components/modules/profiles/Members/MembersList/index.tsx

```diff
@@ -0,0 +1,107 @@
+import { FC, useMemo } from 'react'
+
+import { LoadingState as DefaultLoadingState } from '@baseapp-frontend/design-system'
+
+import { Box, Divider, Typography } from '@mui/material'
+import { usePaginationFragment } from 'react-relay'
+import { Virtuoso } from 'react-virtuoso'
+
+import { UserMembersListFragment } from '../../graphql/queries/UserMembersList'
+import DefaultMemberItem from '../MemberItem'
+import { MemberStatuses, NUMBER_OF_MEMBERS_TO_LOAD_NEXT } from '../constants'
+import { MemberListProps } from '../types'
+
+const MembersList: FC<MemberListProps> = ({
+  meRef,
+  MemberItem = DefaultMemberItem,
+  LoadingState = DefaultLoadingState,
+  LoadingStateProps = {},
+  membersContainerHeight = 400,
+}) => {
+  const { data, loadNext, hasNext, isLoadingNext } = usePaginationFragment(
+    UserMembersListFragment,
+    meRef,
+  )
+
+  const members = useMemo(
+    () =>
+      data?.profile?.members?.edges.filter((edge) => edge?.node).map((edge) => edge?.node) || [],
+    [data?.profile?.members?.edges],
+  )
+
+  const renderLoadingState = () => {
+    if (!isLoadingNext) return <Box sx={{ paddingTop: 3 }} />
+
+    return (
+      <LoadingState
+        sx={{ paddingTop: 3, paddingBottom: 1 }}
+        CircularProgressProps={{ size: 15 }}
+        {...LoadingStateProps}
+      />
+    )
+  }
+
+  const renderMemberItem = (member: any, index: number) => {
+    if (!member) return null
+    if (
+      member.status === MemberStatuses.active &&
+      members[index - 1]?.status !== MemberStatuses.active
+    ) {
+      return (
+        <>
+          <Divider />
+          <MemberItem member={data.profile} memberRole="owner" status={MemberStatuses.active} />
```

Comment:
You can add an if statement inside the return to avoid repeating the following component:
```
<MemberItem
            member={member?.user?.profile}
            memberRole={member?.role}
            status={member?.status}
          />
```